Deploy-CAM.ps1

```diff
@@ -2775,6 +2779,11 @@ function Set-RadiusSettings() {
         }
         $radiusConfig.enableRadiusMfa = $enableRadiusMfa
 
+        if ( $radiusConfig.enableRadiusMfa -and (-not $enableExternalAccess)) {
```

Comment:
That right - I said it wrong in the email. The configuration that has the security issue is when external access is disabled and MFA is enabled. This feels a bit weird to deny it in this way but I think we can live with it for a while.